src/main/java/pl/tomaszdziurko/jvm_bloggers/view/admin/mailing/MailingAddressPage.java

```diff
@@ -0,0 +1,37 @@
+package pl.tomaszdziurko.jvm_bloggers.view.admin.mailing;
+
+import org.apache.wicket.authroles.authorization.strategies.role.Roles;
+import org.apache.wicket.authroles.authorization.strategies.role.annotations.AuthorizeInstantiation;
+import org.apache.wicket.markup.html.basic.Label;
+import org.apache.wicket.markup.repeater.Item;
+import org.apache.wicket.markup.repeater.data.DataView;
+import org.apache.wicket.spring.injection.annot.SpringBean;
+import org.wicketstuff.annotation.mount.MountPath;
+import pl.tomaszdziurko.jvm_bloggers.mailing.domain.MailingAddress;
+import pl.tomaszdziurko.jvm_bloggers.view.panels.CustomPagingNavigator;
+
+@MountPath("mailing-address")
+@AuthorizeInstantiation(Roles.ADMIN)
+public class MailingAddressPage extends AbstractMailingPage {
+
+    public static final int MAILING_ADDRESS_PER_PAGE = 15;
```

Comment:
I think *15* is used for every paging in admin panel, so maybe extract it to some global constant.